drivers/dai/intel/alh/alh.c

```diff
@@ -75,6 +79,10 @@ static void alh_claim_ownership(void)
 	sys_write32(sys_read32(ALHASCTL) | ALHASCTL_OSEL(0x3), ALHASCTL);
 	sys_write32(sys_read32(ALHCSCTL) | ALHASCTL_OSEL(0x3), ALHCSCTL);
 #endif
+#ifdef CONFIG_SOC_SERIES_INTEL_CAVS_V25
+	/* Allow LPGPDMA connection to Audio Link Hub */
+	sys_write32(DSPALHO_ASO_FLAG | DSPALHO_CSO_FLAG, ADSP_DSPALHO_ADDRESS);
+#endif
```

Comment:
what about **alh_release_ownership**? Is it OK to set it permanent for CAVS?

Reply:
Fixed in new revision.